api/src/main/java/org/openmrs/module/fhir2/providers/r3/EncounterFhirResourceProvider.java

```diff
@@ -158,19 +158,19 @@ public IBundleProvider searchEncounter(@OptionalParam(name = Encounter.SP_DATE)
 		        .searchForEncounters(new EncounterSearchParams(date, location, participantReference, subjectReference,
 		                encounterType, tag, hasAndListParam, id, lastUpdated, sort, includes, revIncludes)));
 	}
-	
+
 	/**
 	 * Custom search endpoint that fetches encounters that include medication requests Returns a bundle
 	 * that includes the medication requests and any medication dispenses that reference those requests
+	 * NOTE: requires Core 2.6.1 or higher (with new "declined" fulfiller status)
```

Comment:
Is this comment true?  This method will still work, right?  It just won't exclude declined, but that's ok, because nothing will _be_ declined in pre-2.6.1.

Reply:
Unfortunately, it doesn't work because the underlying Hibernate search needs to map declined to the enum, and it fails when declined doesn't exist.  This is why I needed to move the test for this method to the new 2.7 integration submodule.